README.md

```diff
@@ -1,7 +1,15 @@
 # Orchid
 
-Orchid is a generator which can be used to create a new CDRH API template site.  The new site can either connect to the entire API's contents or filter by "type," meaning that the new site can be used for a specific collection.
+Orchid is a multi-lingual Rails engine which handles common functionality across
```

Comment:
"multi-lingual" is pretty strong for an engine that only has an `en.yml` application :)  Although I've considered that now that we have an `es.yml` translated, maybe we should fold that back into orchid

Reply:
If it covers all the facets that are in Orchid, we may as well :)

Reply:
I think it should!  Then we'd probably need to write some more logic into the generator to copy that file instead of en.yml, but that can be a patch / minor release later, i reckon